### PR TITLE
Document tables with their mostly used fields

### PR DESCRIPTION
**Pull request by szilardhuber**

### CODE REVIEW

File: tables.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+| | *Technical table about Palette internals* |
+
+| `p_serverlogs` ||
+| :---: | --------- |
+| | Loaded from the serverlogs table. This table contains process relationship information. (parent_vizqlsession, parent_dataserver_session etc) and some smaller data cleanup like domain is removed from username and such.|
+| **Notable fields:** |
+| `filename` | The original filename of the log file from which the given line is |
+| `sev` | Severity of the current log line |
+| `k` | Key (or category) of the current log line |
+| `v` | Actual content of the currnet log line |
```
Comment (baspeti) [inline tables.md]: Typo: currnet

File: tables.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+| `elapsed_ms` | Parsed duration of the event descibed in this log line |
```
Comment (baspeti) [inline tables.md]: Typo: descibed

File: tables.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+| **Notable fields:** |
+| `process_id ` | Exact id of the process. This is unique for a given time for a given machine. |
+| `thread_id` | id of the thread if the current measurement is referring to a thread, `-1` otherwise. (`-1` means the current row refers to the whole process.) |
+| `cpu_core_consumption` | The actual cpu usage that happened for this record since the previous measurement. |
+| `memory_usage_bytes` | The actual memory usage in bytes for this record. |
+| `ts_rounded_15_secs` | The timestamp the measurement took place rounded to a 15 seconds polling interval. |
+
+| `plainlogs` ||
+| :---: | --------- |
+| | There two kind of Tableau Server log files. One set of processes log in JSON format and an other set of them just in plain text files. This table contains the raw log lines from plain text log files. |
```
Comment (baspeti) [inline tables.md]: I think it was meant to start like: "There are two kind of ...".
And I don't think I get the second sentence. I guess it was meant to start like: "One set of them processes(? - really? not contains or something else? processes is the right word here?)"

... and I would continue like this: "... and an other set of them is just plain text in files."

File: tables.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+| `serverlogs` ||
+| :---: | --------- |
+| | There two kind of Tableau Server log files. One set of processes log in JSON format and an other set of them just in plain text files. This table contains the raw log lines from JSON log files. |
```
Comment (baspeti) [inline tables.md]: Same comments just as above.

File: tables.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+| `threadinfo` ||
+| :---: | --------- |
+| | Contains raw cpu and memory metric values received from the Tableau Server nodes. |
+
+
+
+| Non-historized repository tables | |
+| :---: | --- |
+|| These tables are polled from the Tableau server node that has the Repository on it. The content of these tables are streamed which means that only changed or newly created records are polled. |
```
Comment (baspeti) [inline tables.md]: I'm also pretty sure that there should be a comma before "which means".

File: tables.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+|| `h_user_default_customized_views` |
+|| `h_users` |
+|| `h_views` |
+|| `h_workbooks` |
+
+| Internally used "technical" tables | |
+| :---: | --- |
+|| These tables are needed for variouos technical reasons. |
+| **Tables:** |
+| This is needed to handle Tableau sessions running around UTC midnight. As our reporting uses UTC midnight as a separator, this is needed to handle sessions that overlap two days. | `cross_utc_midnight_sessions` |
```
Comment (baspeti) [inline tables.md]: I would swap the columns here. So first column would be the table name or pattern, and then the second would be the description. In this case it would be more consistent with the rest of this document.

File: tables.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+| External tables. These tables are used to import data from the csv files sent by the Palette Insight Agents to the database. | `ext_*` |
+| Partitions. Palette Insight utilizes the partitioning feature of Pivotal Greenplum Database and these tables are the actual tables of the partitions. | `_prt_*` |
+| Stage tables. These are loaded during the reporting and after their content is filled that is fully moved to the matching `p_` table.| `s_*` |
```
Comment (baspeti) [inline tables.md]: I would rephrase this sentence like this:

"These are loaded during the reporting, and after their content is filled fully, then its records are moved to the matching `p_` table."

Or something like that.

File: tables.md (hunk 3 above)
Comment (szilardhuber) [reply tables.md]: There are two kinds of Tableau Server log files. One gang, band, group, SET of the processes of that SoaB Tableu are logging in JSON and the other pals log in plain text.

File: tables.md (hunk 7 above)
Comment (szilardhuber) [reply tables.md]: Yeah but fully would me completely different thing that way and that's not what I meant.
